packages/core/src/sharedWishlists/redux/selectors.js

```diff
@@ -143,6 +143,7 @@ const denormalizeSharedWishlistItem = (
  *
  * @param {object} state - Application state.
  * @param {number} sharedWishlistItemId - Numeric identifier of the shared wishlist item.
+ * @param _
```

Comment:
Is this correct?

Reply:
Yes, JSDoc was complaining about it. I even found another case of this in the `getCheckoutDeliveryBundleUpgrade` selector